ansible-st2-local/playbooks/arteriaexpress.yaml

```diff
@@ -1,35 +0,0 @@
----
-- name: Install st2 for arteria
-  remote_user: vagrant
-  hosts: master
-  vars:
-    st2_version: 0.13.2
-    st2web_version: 0.13.2
-  roles:
-    # Postgres requires that the en_US.UTF-8 locale is set - this fixes that
-    - locale
-    - mongodb
-    - rabbitmq
-    - mistral
-    - st2
-    - st2web
-    - arteria
-
-- name: Setup the testarteria1 node
-  remote_user: vagrant
-  sudo: true
```

Comment:
It does. We're not running the `arteriaexpress` playbook for SNP&SEQ purposes though, but the `arteria.yml`, which uses root instead.